multimodal/src/autogluon/multimodal/cli/voc2coco.py

```diff
@@ -98,8 +98,8 @@ def get_image_info(annotation_root, extract_num_from_imgid=True):
     else:
         filename = os.path.basename(path)
     img_name = os.path.basename(filename)
-    if not filename[-4:] in [".jpg", ".png"]:
-        filename = filename + ".jpg"
+    if not img_name[-4:] in [".jpg", ".png"]:
```

Comment:
Just a quick question, how do we know the postfix is definitely .jpg?

Reply:
Added default extension to arguments. This line was a hardcode fix for a dataset's bad annotation file (no extensions for image file names in json annotation file).